corehq/motech/repeaters/tasks.py

```diff
@@ -131,16 +131,19 @@ def process_repeat_record(repeat_record):
         repeat_record.cancel()
         repeat_record.save()
         return
-    if not repeater.is_connection_working():
-        repeater.pause()
-        notify_repeater_admins(repeater)
 
     try:
         if repeater.paused:
             # postpone repeat record by 1 day so that these don't get picked in each cycle and
             # thus clogging the queue with repeat records with paused repeater
             repeat_record.postpone_by(timedelta(days=1))
             return
+
```

Comment:
I'm not sure why the `if` block above is inside this `try` block. I'll follow up and see if we can move both it and the following `if` block out.

But I am keen to get this deployed: A single bad Repeater can cause quite the deluge of notifications.